SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -317,6 +320,32 @@ protected void _userUpdate(Map<String, String> attributes) {
         queueEvent("user", parameters, null);
     }
 
+    protected void _userUpdate (String name, Date date) {
+        Map<String, Object> parameters = new HashMap<String, Object>();
+        Map<String, String> resultAttributes = new HashMap<String, String>();
+
+        String resultantDate = getStringFromDate(date);
+        resultAttributes.put(name, resultantDate);
+
+        try {
+            parameters.put("attributes", new JSONObject(resultAttributes));
+        }catch (NullPointerException ex) {
+            SwrveLogger.e(LOG_TAG, "JSONException when encoding user attributes", ex);
+            return;
+        }
+
+        queueEvent("user", parameters, null);
+    }
+
+    private String getStringFromDate(Date date) {
+
+        TimeZone timezone = TimeZone.getTimeZone("UTC");
+        DateFormat dateformat = new SimpleDateFormat("yyyy-dd-MM'T'HH:mm:ss.SSS'Z'");
```

Comment:
Might be better to initialize this as part of init of sdk? Creating new SimpleDateFormat can be expensive, I think.